### ISSUE TITLE
Multiple images in Card not supported

### ISSUE THREAD

**Issue by sharmadine** [Microsoft/BotBuilder#2437]
HeroCard API accepts & expects a list of images but renders only one image. We have a requirement to show product image in various colors along with 'tap' action to show the selected image.

So, we would need to have a way to show multiple images along with associated actions in a single card. An example below, possible?

![image](https://cloud.githubusercontent.com/assets/2290501/23918809/79c870be-091a-11e7-9145-88136e334fe8.png)

**Comment by mgbennet** [comment]
As you said, hero cards currently support only a single image. I think what you want is a carousel, which takes an array of hero or thumbnail cards, each potentially with their own image, text, and actions, and renders them out side by side, with the ability to scroll through them. You can see an example implementation [here](https://github.com/Microsoft/BotBuilder-Samples/tree/master/Node/cards-CarouselCards). You can't do the exact layout that you have shown, however, since each card will be in a single row.

**Comment by sharmadine** [comment]
Thanks Mike for your response. Yes, carousal works but that's giving a bad experience in our scenario. We have a product with 10+ color variation and showing same product with different color as a card is not so good experience. 

The image that I had shared is from a Kik bot where they are combining multiple images into a single card, each image is linked to its own URL as well as has associated button action.

Since Kik does not support Carousel, this seems a much better option. I am not able to find a pattern to build something like this using botframework. I tried 'html' way of image mapping but HTML page / code can not be embedded in botframework OR I could not find a way to do it.

Any suggestions will be of much help.
